Aptoide/aptoide-client-v8
# [feature/ASV-1477] Category Tops

**What does this PR do?**

   Implements the top list for Games and Apps categories. Additionally it slightly changes the cards corner radius to match the ones on Zeplin.

**Database changed?**

   No

**How should this be manually tested?**

  Check if Top Games and Top Apps work as intended. Namely, check if opening apps in these views work correctly, if they match Zeplin and if the retrieved apps are correct.

**What are the relevant tickets?**

  [ASV-1477](https://aptoide.atlassian.net/browse/ASV-1477)

**Code Review Checklist**
